### DISCUSSION TITLE
Self-signed certificate on webdav

### DISCUSSION THREAD

**Discussion by promexio** [remotely-save/remotely-save#253] [Q&A]
Hi, 
Is it possible to use a self signed cert on a webdav server to pass through https? 
If yes, how? 

cheers and thx

**Comment by fyears** [comment]
sorry but no. no api to achieve that

**Comment by banchaa** [comment]
If you use rooted device with Magisk. Install the CAs to your device. Install Magisk-Module "AlwaysTrustUserCerts" from Github.

Works well :slightly_smiling_face:
